server/db/interface.go

```diff
@@ -256,8 +266,14 @@ type SwapArchiver interface {
 	SaveRedeemB(mid MarketMatchID, coinID []byte, timestamp int64) error
 
 	// SaveRedeemAckSigA records party A's signature acknowledging party B's
-	// redemption.
+	// redemption. Since this is the final match negotiation step, the match is
+	// marked as done/inactive (not the same as archival).
 	SaveRedeemAckSigA(mid MarketMatchID, sig []byte) error
```

Comment:
I just realized this cannot set `active=false` since the ack of the redeemption txn can be provided before the txn reaches the required confirmations for `processBlock` to mark the swap as complete...

Reply:
I've clarified in swap.go comments that `status` and `active` are independent.

https://github.com/decred/dcrdex/blob/63a5445b2802a15d57f0bc6246657ad7bff6b5cb/server/swap/swap.go#L358-L363

The redeem ack sig storage queries are also updated so that either query can set active=false:

https://github.com/decred/dcrdex/blob/63a5445b2802a15d57f0bc6246657ad7bff6b5cb/server/db/driver/pg/internal/matches.go#L127-L134